database/sql/V075__add_metadata_to_quotes_and_order_quotes.sql

```diff
@@ -0,0 +1,11 @@
+-- This migration script is reversible.
+
+-- Step 1: Add two new columns to the quotes table
+ALTER TABLE quotes 
```

Comment:
To keep the data model as stream lined as possible we could make these columns required and fill the historic data with `verified: false` and `metadata: {}`

Reply:
> for historic quotes verified columns value could be wrong

That's true although I'd prefer a cleaner DB model over perfect historic data that will never be read. Also if anybody really cares about the accuracy of that data we know when the migration got applied and can detect historic values with the `{}` metadata value.